docs/src/main/asciidoc/cdi-integration.adoc

```diff
@@ -160,7 +161,7 @@ _Solution_: Use the `AdditionalBeanBuildItem` as described in <<additional_bean_
 == Use Case - I Need To Transform Annotation Metadata
 
 In some cases, it's useful to be able to modify the annotation metadata.
-Quarkus provides a powerful alternative to https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#process_annotated_type[`jakarta.enterprise.inject.spi.ProcessAnnotatedType`, window="_blank"].
+Quarkus provides a powerful alternative to https://jakarta.ee/specifications/cdi/4.0/jakarta-cdi-spec-4.0.html#process_annotated_type[`jakarta.enterprise.inject.spi.ProcessAnnotatedType`, window="_blank"].
```

Comment:
Since we are not implementing CID Full, isn't this now rather a powerful alternative to the `@Enhancement` BCE phase? :)

Reply:
Yeah, I wasn't sure what to do about this. BCE aren't as well known probably, so I'm thinking maybe it's best to mention both?

Reply:
Done.